www.foia.gov/_sass/theme/_typography.scss

```diff
@@ -29,6 +29,14 @@ h2 {
   }
 }
 
+h2 {
+  @extend %heading-2;
+}
+
+.heading-2 {
```

Comment:
I think we should rename this to be specific to its element. We haven't done the best job of sticking to SMACSS (this is mostly on me), but I'd like to avoid introducing other styles as much as possible.